velox/type/Conversions.h

```diff
@@ -393,18 +393,55 @@ template <bool TRUNCATE, bool LEGACY_CAST>
 struct Converter<TypeKind::VARCHAR, void, TRUNCATE, LEGACY_CAST> {
   template <typename T>
   static std::string cast(const T& val) {
-    if constexpr (std::is_same_v<T, double> || std::is_same_v<T, float>) {
-      auto stringValue = folly::to<std::string>(val);
-      if (!FLAGS_experimental_enable_legacy_cast &&
-          stringValue.find(".") == std::string::npos &&
-          isdigit(stringValue[stringValue.length() - 1])) {
-        stringValue += ".0";
-      }
-      return stringValue;
-    }
     return folly::to<std::string>(val);
   }
 
+  static std::string cast(const float& val) {
```

Comment:
Would be nice to add unit tests for these functions.

Reply:
Unit tests are added to TEST_F(CastExprTest, realAndDoubleToString) for the float/real type in  [CastExprTest.cpp](https://github.com/facebookincubator/velox/pull/7602/files#diff-f41a5cc1aa75bd9ae99c7ed93b7a6fe859f3796183fa45969032d5836b9f19e8)

Reply:
It is good to have tests for CAST, but it would be nice to also have unit tests for this specific functionality that doesn't involve CAST experssion.

Reply:
Hi Masha, do you mean unit tests for the function `normalizeStandardNotation()` and `normalizeStandardNotation()`?